sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Support datetime type for keyword argument `accept_datetime` in `get_snapshot_configuration_settings`, `list_snapshot_configuration_settings()` and `list_revisions()`.
```

Comment:
nit: the first function name is missing the parens to be consistent with others in the list

```suggestion
- Support datetime type for keyword argument `accept_datetime` in `get_snapshot_configuration_settings()`, `list_snapshot_configuration_settings()` and `list_revisions()`.
```